hazelcast/src/main/java/com/hazelcast/splitbrainprotection/SplitBrainProtectionOn.java

```diff
@@ -14,25 +14,25 @@
  * limitations under the License.
  */
 
-package com.hazelcast.quorum;
+package com.hazelcast.splitbrainprotection;
 
 /**
- * Represents a cluster quorum type
+ * Represents a cluster split brain protection type
  */
-public enum QuorumType {
+public enum SplitBrainProtectionOn {
```

Comment:
this may be called `SplitBrainProtectioType`.

Reply:
In the configuration we have renamed `type` -> `protectOn` and this is why I've decided to rename it here as well. O know type might look better here but being consistent with the configuration seems more important.